studio/components/interfaces/Settings/Logs/Logs.constants.ts

```diff
@@ -440,36 +440,37 @@ export const PREVIEWER_DATEPICKER_HELPERS: DatetimeHelper[] = [
   {
     text: 'Last hour',
     calcFrom: () => dayjs().subtract(1, 'hour').startOf('hour').toISOString(),
-    calcTo: () => '',
+    calcTo: () => dayjs().toISOString(),
```

Comment:
should not change this, as it would only return logs up to that specific time when page mounts. Pressing refresh will not return any new data.

when the `to` value is an empty string, it basically means "Return all logs from the `from` value all the way to the current time, whatever the current time is"